test/cases/declarations/contexts.hack

```diff
@@ -0,0 +1,7 @@
+function no_listed_contexts(): void {}
+function empty_context()[]: void {}
+function one_context()[C]: void {}
+function many_context()[C1, C2, Cn]: void {}
+function throws_foo_exception()[throws<FooException>]: void {
```

Comment:
I know `throws` is used in the docs, but there isn't an actual type with that name. I suggest this.

```suggestion
function throws_foo_exception()[policied<Foo>]: void {
```